.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+Thanks for contributing to RIOT!
```

Comment:
Maybe some note that this message and the "Preliminary checks" section are supposed to be removed after reading?

Reply:
I was more thinking at adding a message after the checks saying: "if one of the boxes is not checked, we will ignore your contribution" or something more politically correct but with the same meaning

Reply:
I would remove this line, IMHO the template should be short and to the point. Primarily providing structure to help reviewers and testers.

Reply:
Please address @smlng comment